mozilla_django_oidc_db/signals.py

```diff
@@ -0,0 +1,12 @@
+from django.contrib.auth.signals import user_logged_in
+from django.dispatch import receiver
+
+
+@receiver([user_logged_in], dispatch_uid="oidc.set_config_class")
+def set_oidcdb_config_class_on_session(sender, user, request, **kwargs):
+    """
+    Record the OIDC config class on the session, this is needed so the callback view
+    can retrieve the config in case of a SessionRefresh flow
+    """
+    if hasattr(user, "_oidcdb_config_class"):
+        request.session["OIDCDB_CONFIG_CLASS"] = user._oidcdb_config_class
```

Comment:
probably best to explicitly mark this as private with an underscore:

```suggestion
        request.session["_OIDCDB_CONFIG_CLASS"] = user._oidcdb_config_class 
```